test/Project.toml

```diff
@@ -0,0 +1,5 @@
+[deps]
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
It shouldn't be necessary to add these dependencies. But feel free to if it's easier. For `Uniform(200,350)` you can do `200 + 150*rand()`, for `@unpack field = struct` you can do `(;field) = struct` (was introduced in Julia Base in 1.6 or 1.7(?) but I used `@unpack` usually as I find it more explicit.

Reply:
Damn you're right, that's definitely an unnecessary dependency. Will remove the test-specific dependencies.